### DISCUSSION TITLE
Review device_class and state_class

### DISCUSSION THREAD

**Discussion by flauteps** [djansen1987/SAJeSolar#93] [General]
Hi, I'm configuring the energy section. I noticed that some entities do not have the correct device_class. 
For example sensor **nowpower** has **device_class**: "power" but it should be "energy"

![immagine](https://github.com/user-attachments/assets/58864709-9da2-4dc4-800f-86896d25d091)

For more information, see this page
https://www.home-assistant.io/docs/energy/faq/#troubleshooting-missing-entities

**Comment by djansen1987** [comment]
Thanks for your feedback, are you able to create an pull request with the updated code? Please do test it before committing the code

**Reply by djansen1987** [reply]
it is merged and tested, thank you for contributing and your donation!
